src/Raven.Server/Documents/Handlers/BatchHandler.cs

```diff
@@ -652,9 +652,10 @@ protected override long ExecuteCmd(DocumentsOperationContext context)
                                             }
                                         }
 
+                                        var document = cmd.Document.Clone(context);
                                         var putResult = Database.DocumentsStorage.Put(context, cmd.Id, null, cmd.Document.Clone(context), changeVector: changeVector,
```

Comment:
shouldn't you use that here? I see another `cmd.Document.Clone(context)` as on of the arguments

Reply:
changed